pkg/authorization/workspace_content_authorizer.go

```diff
@@ -178,22 +233,43 @@ func (a *workspaceContentAuthorizer) Authorize(ctx context.Context, attr authori
 			}
 		}
 		if len(errList) > 0 {
+			kaudit.AddAuditAnnotations(
+				ctx,
+				WorkspaceContentAuditDecision, DecisionNoOpinion,
+				WorkspaceContentAuditReason, fmt.Sprintf("errors from parent authorizer: %v", utilerrors.NewAggregate(errList)),
+			)
 			return authorizer.DecisionNoOpinion, strings.Join(reasonList, "\n"), utilerrors.NewAggregate(errList)
 		}
 	}
 
 	// non-admin subjects don't have access to initializing workspaces.
 	if ws.Status.Phase == tenancyv1alpha1.ClusterWorkspacePhaseInitializing && !extraGroups.Has(bootstrap.SystemKcpClusterWorkspaceAdminGroup) {
+		kaudit.AddAuditAnnotations(
+			ctx,
+			WorkspaceContentAuditDecision, DecisionNoOpinion,
+			WorkspaceContentAuditReason, "only admin subjects may access initializing workspaces",
```

Comment:
same as above, this is a possible resolution, the reason is rather something like `toplevel.authorization.kcp.io/reason: not permitted, clusterworkspace is in initializing phase`.